README.md

```diff
@@ -118,6 +120,37 @@ Also tests can be filtered using `make test <filter>`:
 $ make test wpcom.site.post
 ```
 
+## Browser Tests
+
+To run the test suite in a browser, you will need to have `serve` installed globally:
```

Comment:
We should be able to define `serve` in `devDependencies`, then change the `run-test-app` Make target to run `serve` from `./node_modules/.bin/serve` , thereby avoiding the need for this additional step.